tests/foreman/api/test_location.py

```diff
@@ -78,7 +78,6 @@ def make_entities(self):
         return dict(
             domain=entities.Domain().create(),
             subnet=entities.Subnet().create(),
-            env=entities.Environment().create(),
```

Comment:
@vsedmik do you think this would affect puppet-related tests?

Reply:
I don't think it would. AFAIK we don't use this fixture on pp-enabled sat.